### PR TITLE
feat(PaymentCard): adds card_status not_active

### LINKED ISSUE #2572: Add support for new card status: "not active" / "ikke aktivert"
## 🚀 Feature Proposal

An Update to existing feature

Using the Eufemia library for cards at work. And want to put cards in a "Not activated" state. I looked through the code and it seems like there are only "Active, blocked and expired" states. Currenly, the states does not quite match the api, as we also get the cardStatus "NOT_ACTIVE".

Need to add the not active state styling in to match the api
Please outline the motivation for the proposal.

For each "not activated" card status, the card will be using that styling.

We are currently using the PaymentCard extension, as there was no other "cards" components in eufemia other than the PaymentCard, and need support for this cardStatus

### PR DESCRIPTION
**Pull request by dsa015**
## Summary

Link to issue: https://github.com/dnbexperience/eufemia/issues/2572

Add support for new card status "not_active" as the existing states does not match api.
image

Compact version:
image

This will allow us to add the styling for cards that are not activated yet. 

Closes #2572

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/payment-card/icons/CardInMedium.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+const CardInMedium = (props) => (
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/payment-card/icons/CardInMedium.tsx:2]:
```suggestion
const CardIn = (props) => (
```

We could perhaps rename/replace all references of `CardInMedium` with `CardIn` 🤔 
I think this fits better with the existing icons in PaymentCard, as they don't seem to be suffixed with the icon size(like medium).

**Review by langz — COMMENTED** [review]
Thanks for the great contribution @dsa015 🚀 
This all looks very good, and looks to align very well with the existing `PaymentCard` code 👏 
Just added one comment/suggestion.

By the way, in which application is this functionality needed? Just me being curious 😊

**Comment by dsa015** [comment]
Hello, thank you for the feedback! 😁 Its my first ever contribution to an open source project so I was very excited for this 😎

We are using this extension in our web app for each users cards overview page. coming from CB department
I will also change the CardInMedium to CardIn!

**Comment by langz** [comment]
> 

Good good, great work 👍 
I think we can include this in release v10.7 #2571 

I'll approve this PR, squash the commits, and change the commit message to follow our [Git conventions](https://eufemia.dnb.no/contribute/style-guides/git/#commit-messages) 💯

**Comment by dsa015** [comment]
Great, thank you! 😄

**Pull request merged**